e2e_samples/unstructured_data/src/experiments/llm_citation_generator/main.py

```diff
@@ -27,53 +28,63 @@ def __init__(
         question: str,
         user_prompt_file: str = "user_prompt.txt",
         system_prompt_file: str = "system_prompt.txt",
-        run_id: Optional[str] = None,
-        variant_name: Optional[str] = None,
         db_question_id: Optional[int] = None,
         token_encoding_name: str = "o200k_base",  # gpt-4o, see https://gpt-tokenizer.dev/ for encoding names
         max_tokens: int = 4096,  # Azure gpt-4o max tokens defaults to 4096
         overlap: int = 100,
+        submission_container: str = "msft-quarterly-earnings",
+        results_container: str = "msft-quarterly-earnings-di-results",
+        run_id: Optional[str] = None,
         **kwargs: Any,
     ) -> None:
-        self.question = question
+        # Fetch config values from env
+        fetcher = EnvFetcher()
```

Comment:
this loads the config values needed to run the generator so we fail early when variables are not set. The fetcher is hardcoded to pull from env for now but can be updated to fetch from Key Vault if we ever need that in the future